cmd/kogito/command/install/data_index.go

```diff
@@ -165,7 +167,7 @@ func (i *installDataIndexCommand) Exec(cmd *cobra.Command, args []string) error
 		i.flags.infinispan.UseAuth = true
 		i.flags.infinispan.SaslMechanism = v1alpha1.InfinispanSaslMechanismType(i.flags.infinispanSasl)
 		if err := kubernetes.ResourceC(i.Client).Create(&infinispanSecret); err != nil {
-			return fmt.Errorf("Error while trying to create an Infinispan Secret credentials: %s", err)
+			return fmt.Errorf("Error while trying to create an Infinispan Secret credentials: %s ", err)
```

Comment:
is all of those extra spaces above intentional?

Reply:
Yes, the go lint advice add a space after the format var